portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,94 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
+
   </head>
   <body>
+    <div id="header">
```

Comment:
FYI: the css crops the bottom of the header text, at least on my browser.

<img width="462" alt="Screen Shot 2020-05-28 at 1 57 50 PM" src="https://user-images.githubusercontent.com/65307754/83193034-409cd480-a0eb-11ea-9e3b-ae7505c6e5b4.png">

Reply:
Oh weird, I'll try to fix that. Thanks!